tests/integration/app.test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * https://github.com/electron/spectron#properties
+ * https://jestjs.io/docs/en/expect
+ */
+
+const Application = require('spectron').Application;
+
+const appPAth = process.env.TESTING_APP_PATH;
+let app;
+
+describe('Application launch', () => {
+  beforeEach(async () => {
+    app = new Application({
+      path: appPAth,
+    });
+    await app.start();
+    return app.client.waitUntilWindowLoaded();
+  });
+
+  afterEach(() => {
+    if (app && app.isRunning()) {
+      return app.stop();
+    }
+    return false;
+  });
+
+  test('should test true is true', async () => { // true
+    expect(true).toBe(true);
+  });
```

Comment:
Possibly noob question, What would be the purpose of this?

Reply:
was there to test that the test were running properly. can remove it.